sdk/react/panels/EndScreen.js

```diff
@@ -69,10 +69,11 @@ var EndScreen = React.createClass({
   },
 
   _renderDefaultScreen: function(progressBar, controlBar) {
-    var fullscreenPromoImage = (this.props.config.endScreen.mode == 'default');
+    var endScreenConfig = this.props.config.endScreen || {};
+    var fullscreenPromoImage = endScreenConfig === 'default';
```

Comment:
I notice the interchanging of single `''` and double `""` quotes throughout. Is there a reason for this?